hdf/src/dfsd.c

```diff
@@ -4673,7 +4672,6 @@ DFSDwriteslab(int32 start[], int32 stride[], int32 count[], void *data)
     int32 *filedims;   /* tmp array containing the dimensions */
                        /*   of the dataset in the file */
     int32 r_error;     /* flag if an error occurred, */
-                       /*   used by DFconvert macro */
```

Comment:
I just removed the "used by DFconvert macro" lines. Everything else is reformatting.